backend/spiritual_avatar_generation_engine.py

```
@@ -392,70 +242,36 @@ def _get_voice_settings(self, voice_tone: str) -> Dict[str, float]:
+        async with self.db.get_connection() as session:
+            try:
+                new_session = AvatarSession(
+                    session_id=session_id, user_email=user_email, guidance_text=guidance_text,
+                    avatar_style=avatar_style, voice_tone=voice_tone, video_url=video_url,
+                    audio_url=audio_url, duration_seconds=duration_seconds, voice_cost=voice_cost,
+                    video_cost=video_cost, total_cost=voice_cost + video_cost,
+                    generation_time=generation_time, status='completed'
+                )
+                session.add(new_session)
+                await session.commit()
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)